reference/robot.md

```diff
@@ -21,9 +21,11 @@ Direct derived nodes: [DifferentialWheels](differentialwheels.md), [Supervisor](
 
 ### Description
 
-The [Robot](#robot) node can be used as basis for building a robot, e.g., an articulated robot, a humanoid robot, a wheeled robot...
+The [Robot](#robot) node can be used as basis for building a robot, e.g., an articulated robot, a humanoid robot, a wheeled robot.
 If you would like to build a robot with supervisor capabilities use the [Supervisor](supervisor.md) node instead (Webots PRO license required).
 
+> **Note**: If the Robot node has one or more Solid (or derived) ancestor nodes, then the physical properties of the ancestor nodes will affect the Robot node physical behavior.
```

Comment:
I would this sentence with "Obviously": "Obviously, if the Robot node..."

Reply:
"Obviously" is very harsh like this. I would say "Logically".